MvvmCross/Core/Binding/BindingContext/MvxFluentBindingDescription.cs

```diff
@@ -124,6 +125,11 @@ public MvxFluentBindingDescription(IMvxBindingContextOwner bindingContextOwner,
             return WithConversion(converterName, converterParameter);
         }
 
+        public MvxFluentBindingDescription<TTarget, TSource> WithDictionaryConversion<TFrom, TTo>(Dictionary<TFrom, TTo> converterParameter)
```

Comment:
I'm not so sure about having a method in `MvxFluentBindingDescription` that simply acts as an alias to a specific `ValueConverter`. We have the exception of `CommandParameter`, but I don't think this converter is as important.

Reply:
Do you feel it should be moved, possibly to the `MvxFluentBindingDescriptionExtensions` class or that it should be removed?

Reply:
Yeah, moving it to the extensions that are on the same namespace would be super nice 👌